zuul/layout.yaml

```diff
@@ -126,6 +144,11 @@ projects:
       - gate-networking-cisco-dsvm-tempest-smoke-asr1k-ocata-centos-7-asr
       - gate-networking-cisco-dsvm-tempest-smoke-asr1k-newton-centos-7-asr
       - gate-networking-cisco-dsvm-tempest-smoke-asr1k-mitaka-centos-7-asr
+    experimental-asr-prll:
+      - gate-networking-cisco-dsvm-tempest-smoke-asr1k-prll-ocata-centos-7-asr-prll
+      - gate-networking-cisco-dsvm-tempest-smoke-asr1k-prll-pike-centos-7-asr-prll
+      - gate-networking-cisco-dsvm-tempest-smoke-asr1k-prll-newton-centos-7-asr-prll
```

Comment:
These new jobs should be adding the dict format to reduce the resources used by the experimental job:

```
- gate-networking-cisco-dsvm-tempest-smoke-asr1k-prll-ocata-centos-7-asr-prll:
  - gate-networking-cisco-dsvm-tempest-smoke-asr1k-prll-pike-centos-7-asr-prll
  - gate-networking-cisco-dsvm-tempest-smoke-asr1k-prll-newton-centos-7-asr-prll
```

They should also be adding to the normal experimental queue and marked non-voting, for an example on how to do that see the ironic-cimc-nexus job definition.

Reply:
Agree but since these job names will only live as long as it takes to get parallel execution working I want them to immediately kick off in parallel to test.